openedx/core/djangoapps/programs/utils.py

```diff
@@ -226,12 +264,38 @@ def completed_course_runs(self):
         Returns:
             list of dicts, each representing a course run certificate
         """
+        return self.course_runs_with_state['completed_runs']
```

Comment:
it seems strange to me to do a reformatting of failed_course_runs in a method that is not related to it. what is the downside of reformatting that data in a different property?